app/components/Navigation.js

```diff
@@ -30,9 +39,7 @@ class Navigation extends Component {
                 </span>
               }
             >
-              <NavDropdown.Item eventKey="4.1" href={`/profile`}>
-                Profile
-              </NavDropdown.Item>
+              <NavDropdown.Item to={`/profile`}>Profile</NavDropdown.Item>
```

Comment:
the link to Profile page is broken. I'll take a look at the link to NavDropDown.Item tmr.

Reply:
Oh I see. It looks like NavDropdown.Item doesn't support a to attribute. It wants href. But since we can't use href, we're going to need to use a different html element here

Reply:
Just fixed it.